src/evaluators.js

```diff
@@ -72,7 +76,10 @@ export function createSafeEvaluatorFactory(
   unsafeRec,
   safeGlobal,
   transforms,
-  sloppyGlobals
+  sloppyGlobals,
+  rejectHtmlComments,
+  rejectImportExpressions,
+  rejectSomeDirectEvalExpressions
```

Comment:
Once an options list gets this long, it should no longer be treated positionally. Better to put all four (five?) of these onto an options-like object. This issue arises several places.